.github/workflows/build-sw-container.yml

```
@@ -38,8 +38,13 @@ jobs:
         TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
       run: |
         docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
-        tag_list=$docker_repo:$REPO,$docker_repo:$REPO-$TIMESTAMP
-        echo "::set-output name=taglist::$tag_list"
+        tag_list=()
+        for registry in hub.opensciencegrid.org/ ''; do
+          for image_tag in "$REPO" "$REPO-$TIMESTAMP"; do
+            tag_list+=("$registry$docker_repo":"$image_tag")
+          done
+        done
```

```suggestion
        for registry in hub.opensciencegrid.org docker.io; do
          for image_tag in "$REPO" "$REPO-$TIMESTAMP"; do
            tag_list+=("$registry/$docker_repo":"$image_tag")
          done
        done
```
pedantic style thing; feel free to ignore